<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test11_2AClass</class>
 <widget class="QMainWindow" name="Test11_2AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test11_2A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Rectangle" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QToolButton" name="btnRect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>37</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="test11_2a.qrc">
       <normaloff>:/Test11_2A/Resources/rectangle.png</normaloff>:/Test11_2A/Resources/rectangle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnCircle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>37</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="test11_2a.qrc">
       <normaloff>:/Test11_2A/Resources/circle.png</normaloff>:/Test11_2A/Resources/circle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnTriangle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>37</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="test11_2a.qrc">
       <normaloff>:/Test11_2A/Resources/triangle.png</normaloff>:/Test11_2A/Resources/triangle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Rectangle</class>
   <extends>QFrame</extends>
   <header>Rectangle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="test11_2a.qrc"/>
 </resources>
 <connections/>
</ui>
